.github/ISSUE_TEMPLATE/unicodedata.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feedback on Unicode data, properties, algorithms
+about: Report bugs and suggestions elsewhere
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## If you have access to the Unicode-internal PAG repo
+Please submit a PAG issue there.
```

Comment:
Should this instead be "for members of the PAG WG…" or some other wording? Otherwise, probably fine.

Reply:
Not just members of PAG. Other Unicoders with PAG triage access sometimes also submit PAG issues directly.

Reply:
Something like:
> Members of the PAG WG and others with access ...

That covers it from both ends.